keras-onnx/onnx_keras/frontend.py

```diff
@@ -221,7 +221,6 @@ def convertToOnnx(self, optimize=0, input_shape=None):
       )
     # Create the model (ModelProto)
     self.omodel = O.helper.make_model(graph_def, producer_name='Kneron')
-    self.omodel.opset_import[0].version = 9
```

Comment:
We may set it to 11 instead of removing it.

Reply:
Sure, let me add it back